src/integrations/prefect-redis/tests/test_messaging.py

```diff
@@ -346,6 +358,65 @@ async def handler(message: Message):
         assert message.attributes == {"id": str(i)}
 
 
+async def test_trimming_streams(
+    redis: Redis, publisher: Publisher, consumer_a: Consumer, consumer_b: Consumer
+) -> None:
+    """Test that streams are trimmed after messages are processed."""
+    # Given the consumers an aggressive trimming frequency for the tests
+    consumer_a.trim_every = timedelta(seconds=0)
+    consumer_b.trim_every = timedelta(seconds=0)
+
+    # Make sure we're starting with a clean slate
+    assert await redis.xlen("message-tests") == 0
+
+    # Publish a known number of messages to the stream
+    TO_SEND = 113
+    async with publisher as p:
+        for i in range(TO_SEND):
+            await p.publish_data(b"hello, world!", {"sequence": f"{i + 1}"})
+
+    assert await redis.xlen("message-tests") == TO_SEND
+
+    # ...then run two consumers that will each see some of the messages, capturing the
+    # ones they've seen along the way
+    seen_messages = {
+        "A": set(),
+        "B": set(),
+    }
+
+    async def handler(consumer_name: str, message: Message):
+        sequence = int(message.attributes["sequence"])
+        seen_messages[consumer_name].add(sequence)
+        total_seen = sum(len(seen) for seen in seen_messages.values())
+        logging.getLogger("prefect.tests").info(
```

Comment:
nit
```suggestion
        logging.getLogger("prefect_redis.tests").info(
```

Reply:
woops this is just left from debugging, ty